submissions/AsyaYeromina/dom-api/script.js

```diff
@@ -0,0 +1,75 @@
+    },
+    {
+        text :`Electro (or electro-funk) is a genre of electronic music and early hip hop directly influenced by the use of the Roland TR-808 drum machines, and funk. Records in the genre typically feature drum machines and heavy electronic sounds, usually without vocals, although if vocals are present they are delivered in a deadpan manner, often through electronic distortion such as vocoding and talkboxing. This is the main distinction between electro and previously prominent genres such as disco, in which the electronic sound was only part of the instrumentation.`,
+        tabName : "Electro",
+        audio: `<iframe width="100%" height="450" scrolling="no" frameborder="no" allow="autoplay" src="https://w.soundcloud.com/player/?url=https%3A//api.soundcloud.com/playlists/819362814&color=%23ff5500&auto_play=false&hide_related=false&show_comments=true&show_user=true&show_reposts=false&show_teaser=true"></iframe><div style="font-size: 10px; color: #cccccc;line-break: anywhere;word-break: normal;overflow: hidden;white-space: nowrap;text-overflow: ellipsis; font-family: Interstate,Lucida Grande,Lucida Sans Unicode,Lucida Sans,Garuda,Verdana,Tahoma,sans-serif;font-weight: 100;"><a href="https://soundcloud.com/digitalstreams" title="Westcoast Radio (EDM)" target="_blank" style="color: #cccccc; text-decoration: none;">Westcoast Radio (EDM)</a> · <a href="https://soundcloud.com/digitalstreams/sets/electroworkout" title="Best Electro of All-Time (Vol. 01)" target="_blank" style="color: #cccccc; text-decoration: none;">Best Electro of All-Time (Vol. 01)</a></div>`
+    },
+]
+
+const buttonList = document.querySelector(".menu__tab-list");
+const contentWrapper = document.querySelector(".content__wrapper");
+const contentList = document.querySelector(".content__list");
+
+
+
+function initApp() {
+    generateHTMLButtons();
+    buttonList.addEventListener("click", contentToggle);
+    layoutSetup();
+}
+initApp();
+
+function generateHTMLButtons() {
+    let HTMLstring = "";
+
+    for (let i = 0; i < tabContent.length; i++) {
+        const tabItem = tabContent[i];
+        const tabName = tabItem["tabName"];
+        HTMLstring += `<li class="menu__tab-item"><button data-id="${i}" class="menu__tab-button">${tabName}</button></li>`;
+    }
+    buttonList.insertAdjacentHTML("beforeend", HTMLstring);
+}
+
+function layoutSetup() {
+    document.querySelector(".menu__tab-button").classList.add("menu__tab-button--active");
+    contentWrapper.insertAdjacentHTML("beforeend", tabContent[0]["text"]);
```

Comment:
`tabContent[0].text` maybe? No need for square brackets here (there are similar cases later in the code)

Reply:
Thanks for remark. I've fixed all square brackets